cdc/sink/mysql/txn_cache.go

```diff
@@ -35,17 +35,23 @@ func (t *txnsWithTheSameCommitTs) Append(row *model.RowChangedEvent) {
 			zap.Any("row", row))
 	}
 	if t.txns == nil {
-		t.txns = make(map[model.Ts]*model.SingleTableTxn)
+		t.txns = make([]*model.SingleTableTxn, 0)
```

Comment:
`nil` is considered a valid slice in Go. No need to create it by `make`. `append` handles the `nil` case automatically.

Reply:
Good suggestion, thanks for your review.